_articles/use-openvpn.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: article
+title: Use OpenVPN 
+description: >
+  This page will go over setting up and using OpenVPN to browse the internet more securely in places like airports, cafe shops and more.
+keywords:
+  - OpenVPN
+  - Security
+  - Privacy
+  - ProtonVPN
+image: http://support.system76.com/images/system76.png
+hidden: true
+section: articles
+
+---
+
+# **Use OpenVPN**
+
+For all Desktop Enviroments, Window Managers and anything in general you will need the 'network-manager-openvpn' package, this will also install the openvpn package as that is a dependency.
```

Comment:
Please spell check.  Also, please use the style guide.  Program names are underlines.